app/src/test/kotlin/org/cru/godtools/ExternalSingletonsModule.kt

```diff
@@ -86,6 +88,7 @@ class ExternalSingletonsModule {
     val languagesRepository: LanguagesRepository by lazy {
         mockk {
             every { getLanguagesFlow() } returns flowOf(emptyList())
+            every { findLanguageFlow(any()) } returns flowOf(Language(Locale.ENGLISH))
```

Comment:
let's default this behavior to `flowOf(null)`, leave it as the responsibility of individual tests using this mock to define expected objects